src/stylesheets/global.scss

```diff
@@ -6,7 +6,7 @@
 
 body {
     margin: 0;
-    padding:  55px 0 0 0 ;
+    padding:  49.188px 0 0 0 ;
```

Comment:
0.188px is a little too specific...
It does work with 50px tho, so that would be more readable

Reply:
Also, I'd love to know how you came up with this number. Did you use an editor?

Reply:
50px will still have a white line if you go smaller like 320px.
<img width="259" alt="Screen Shot 2020-03-05 at 12 02 16 PM" src="https://user-images.githubusercontent.com/33134794/76021065-6e51f300-5ed9-11ea-915d-a5a72974c17b.png">
If you want readable then change the navbar height to be 50px and body 50px will resolve the line and keep the values readable.

Reply:
Let's go with your suggestion then, it sounds much better than leaving it at 49.188px. Make the change and I'll merge your PR

Reply:
Google chrome inspector. It shows you the dimensions of elements. That's how I saw the navbar height is exactly 49.188px at 320px. Only problem is I have to add .812 of padding to the navbar.

Okay, I think I found a better solution. Pushing changes to review.